jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb

```diff
@@ -178,8 +181,7 @@
    "outputs": [],
    "source": [
     "# Define the Subscripton UUID\n",
-    "#subscription_id = \"635c57d3-b921-453f-8e79-70929e50a470\" # Monthly Building Construction\n",
-    "subscription_id = \"8869cf69-9596-47ab-8d94-52b94fcf528e\" # Monthly Road Construction\n",
+    "subscription_id = \"PASTE_YOUR_SUBSCRIPTION ID HERE\"\n",
```

Comment:
you may want to show an example of hitting the subscriptions list endpoint to see all available subscriptions in case the notebook user doesn't have a subscription id on hand

Reply:
Hmm, thats a good point; maybe I'll keep those two commented out subscriptions present as potential examples.

Reply:
FYI just made this change and pushed it to the PR.